exercises/practice/pascals-triangle/.meta/lib/example.dart

```diff
@@ -1,24 +1,23 @@
 class PascalsTriangle {
   List<List<int>> rows(int rows) {
-    if (rows <= 0) {
-      return [];
-    } else {
-      List<List<int>> triangle = new List<List<int>>();
+    /// Row that was last inserted in the triangle.
+    /// This also adds stateful behaviour to the closure passed to List.generate.
+    List<int> triangle = [];
```

Comment:
I think thats actually wrong, you can use `previousRow` instead of `lastRow`, because the triangle is what is being returned from the 
`return List.generate` statement.

Additionally you can dothr following if you want to make it clear how triangle is being generated.

```dart
final triangle = List.generate...
...
...
return triangle;
```

Reply:
Okay I will incorporate it

Reply:
Okay I will incorporate it